kong/db/strategies/cassandra/init.lua

```diff
@@ -226,6 +230,8 @@ local function get_query(self, query_name)
 end
 
 
+local serialize_foreign_pk
```

Comment:
This forward declaration looks unnecessary (it would be useful if we were supporting composite FKs in PKs, but as it stands it looks like a leftover).